.vscode/tasks.json

```diff
@@ -7,7 +7,19 @@
             "type": "process",
             "args": [
                 "build",
-                "${workspaceFolder}/src/OctoshiftCLI/OctoshiftCLI.csproj",
+                "${workspaceFolder}/src/gei/gei.csproj",
```

Comment:
we either need a build task that builds the entire solution or we need to have 3 build talks, 1 for the solution, 1 for `gei` and 1 for `ado2gh`.  I almost always tend to build the entire solution since it's very fast so I am personally leaning toward the first approach (having one build task for the entire solution).

Reply:
@ArinGhazarian So, currently there are 0 tasks that work. My goal was to make one that worked. Can we merge this as is and iterate with a future PR?

Reply:
yeah np. we can always iterate later. [Here](https://wakeupandcode.com/debugging-multiple-net-core-projects-in-vs-code/) I found an example of how to configure launch setting for multiple projects.